### DISCUSSION TITLE
Vuejs+ pocketbase

### DISCUSSION THREAD

**Discussion by matii3** [pocketbase/pocketbase#767] [Q&A]
I am trying to connect vuejs with pocketbase by creating a user in vuejs using pocketbase has a front end. I moved the pocketbase folder into my vuejs project folder. And ran the following commands in sign up vue.

 import {PocketBase} from 'pocketbase';

const client = new PocketBase('http://127.0.0.1:8090');
 
And when i go to vue.js on the web brower i get the following the errors

 ERROR  Failed to compile with 1 error                                                                                                                                                                     19:31:20

 error  in ./src/components/SignUp.vue?vue&type=script&lang=js

Module not found: Error: Can't resolve 'pocketbase' in 'C:\Users\matie\onlytry\src\components'

ERROR in ./src/components/SignUp.vue?vue&type=script&lang=js (./node_modules/babel-loader/lib/index.js??clonedRuleSet-40.use[0]!./node_modules/vue-loader/dist/index.js??ruleSet[0].use[0]!./src/components/SignUp.vue?vue&type=script&lang=js) 2:0-40

can someone please assist me in resolving this issue?

**Comment by ganigeorgiev** [comment]
`PocketBase` is the default export, aka. you don't need the curly brackets around the import statement:

```js
import PocketBase from 'pocketbase';

const client = new PocketBase('http://127.0.0.1:8090');
```

More information you could find in https://github.com/pocketbase/js-sdk#installation

**Reply by matii3** [reply]
I tried without the curly brackets and I am getting the same error. Thank you let me read through the link you sent

**Reply by ganigeorgiev** [reply]
Did you install the package before that?

```sh
npm install pocketbase --save
```

**Reply by matii3** [reply]
Do i run the above command in the command line with my vue application?

**Reply by ganigeorgiev** [reply]
@matii3 I'm not sure that I understand what do you mean. From the error, I guess you are using a bundler (eg. vite) and npm, right? Then you have to install the PocketBase JavaScript SDK the same way as any other js package. For example, if you are following the vue guide from here - https://vuejs.org/guide/quick-start.html#creating-a-vue-application, you have to do the following:

```sh
# initialize your vue project
npm init vue@latest

# navigate to the created project directory
cd <your-project-name>

# install the main dependencies
npm install

# install the PocketBase JS SDK
npm install pocketbase --save

# run your project
npm run dev
```
